src/index.ts

```diff
@@ -1,5 +1,11 @@
-import ChromedriverLauncher from './launcher';
-import ElectronWorkerService from './service';
+import ChromedriverLauncher from './launcher.js';
+import ElectronWorkerService from './service.js';
 
 export default ElectronWorkerService;
 export const launcher = ChromedriverLauncher;
+export type electronService = {
+  api: (...arg: unknown[]) => Promise<unknown>;
+  app: (funcName: string, ...arg: unknown[]) => Promise<unknown>;
+  mainProcess: (funcName: string, ...arg: unknown[]) => Promise<unknown>;
+  browserWindow: (funcName: string, ...arg: unknown[]) => Promise<unknown>;
+} & WebdriverIO.Browser;
```

Comment:
It seems like we should extend the `WebdriverIO.Browser` interface to add these to it, e.g. like we do with the [`@wdio/shared-store-service`](https://github.com/webdriverio/webdriverio/blob/main/packages/wdio-shared-store-service/src/index.ts#L10-L22). Wdyt?

Reply:
I tried something like this, also went for a re-export of browser from `@wdio/globals` as I read wdio is moving away from globals.  Works well I think